lib/bambora/client.rb

```diff
@@ -163,7 +163,7 @@ def bank_profiles(api_key:)
     #
     # @param api_key [String] API key for the bank profiles endpoint.
     #
-    # @return [Bambora::Bank::PaymentProfileResource]
+    # @return [Bambora::Bank::BatchReportResource]
     def batch_reports(api_key:)
       @batch_reports = Bambora::Bank::BatchReportResource.new(
```

Comment:
Change here as you can see from the line 168 the expected return is `Bambora::Bank::BatchReportResource` not `Bambora::Bank::PaymentProfileResource`